### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by AbdifatahYasin1**
### Form validation

This branch is created to validate my portfolio form.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  text-align: center;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  font-weight: bolder;
+  width: 30%;
+  margin-left: 15px;
+  margin-bottom: -20px;
+  font-size: 17px;
+  letter-spacing: 3%;
+}
```
Comment (franclobo) [inline style.css:134]: - [ ] kindly align the button to the left to look similar to the Figma template. 👇🏽 

| **Figma template** | **Your template** |
|---|---|
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    <div class="container-aligment">
+  <h2>Contact me</h2>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+</div>
+<div class="form">
+  <form action="https://formspree.io/f/xgeqkbbo" method="post">
+    <input type="text" placeholder="Alexandra Loremipsum" maxlength="30" required>
+    <input type="email" placeholder="alexandra@gmail.com" required>
+    <textarea type="text" name="message" id="" cols="30" rows="10" placeholder="write your message here" maxlength="500" required></textarea>
+    <button type="submit" class="big-button big-button-2">
```
Comment (franclobo) [inline index.html]: - [ ] Please 🙏🏽 review that the button works as expected for a better user experience.

https://user-images.githubusercontent.com/58642949/192792483-c60eb4dd-c161-42ac-b212-b3ad1eb0c4c4.mp4

File: index.html [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+    <!-- Contact section -->
+ <section class="contact">
+  <div class="container-contact">
+    <div class="container-aligment">
+  <h2>Contact me</h2>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+</div>
+<div class="form">
+  <form action="https://formspree.io/f/xgeqkbbo" method="post">
+    <input type="text" placeholder="Alexandra Loremipsum" maxlength="30" required>
```
Comment (franclobo) [inline index.html]: - [ ] Kindly use your second name instead of "Loremipsum" for a better user interface. 👍🏽

File: style.css [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+/* Conatact */
+.contact {
+  background-image: url(images/Contact\ me.png);
+  background-repeat: no-repeat;
+  flex-wrap: wrap;
+  max-width: 400px;
+  margin: 0 auto;
+  padding: 40px 30px;
+  text-align: right left;
+}
```
Comment (franclobo) [inline style.css]: - [ ] Please 🙏🏽 make sure that you are using Flexbox `display: flex `to lay out the elements of the page. It is a requirement:

> In order to lay out the elements on the page you should use Flexbox.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @AbdifatahYasin1 and @Daniel-Kigozi, 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add the contact information of your partner, this is a pair programming activity. 👇🏽 
image

- [ ] Please 🙏🏽 add a meaningful title and a summary of the activities in your pull request to document your work professionally.

> Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @AbdifatahYasin1 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**